odl/operator/operator.py

```diff
@@ -93,42 +77,247 @@ def _default_apply(self, x, out, *args, **kwargs):
     -------
     `None`
     """
-    out.assign(self._call(x, *args, **kwargs))
+    out.assign(op._call_out_of_place(x, **kwargs))
 
 
-class _OperatorMeta(type):
+def _signature_from_spec(func):
```

Comment:
You are correct, it should. But in that case the functions wont work AT_ALL unless `super().__init__` has been called. Perhaps it is better?